webpack/components/extensions/HostDetails/Tabs/ModuleStreamsTab/ModuleStreamsTab.js

```diff
@@ -4,19 +4,81 @@ import { translate as __ } from 'foremanReact/common/I18n';
 import { Skeleton, Label, Hint, HintBody, Button } from '@patternfly/react-core';
 import PropTypes from 'prop-types';
 import { upperFirst } from 'lodash';
-import { TableText, TableVariant, Thead, Tbody, Tr, Th, Td } from '@patternfly/react-table';
+import { TableText, TableVariant, Thead, Tbody, Tr, Td } from '@patternfly/react-table';
 import {
   LongArrowAltUpIcon,
   CheckIcon,
 } from '@patternfly/react-icons';
 import { urlBuilder } from 'foremanReact/common/urlHelpers';
 import { selectModuleStreamStatus, selectModuleStream } from './ModuleStreamsSelectors';
-import { useBulkSelect, useUrlParams } from '../../../../Table/TableHooks';
+import { useBulkSelect, useTableSort, useUrlParams } from '../../../../Table/TableHooks';
 import { getHostModuleStreams } from './ModuleStreamsActions';
 import InactiveText from '../../../../../scenes/ContentViews/components/InactiveText';
 import TableWrapper from '../../../../../components/Table/TableWrapper';
 import hostIdNotReady from '../../HostDetailsActions';
 import { selectHostDetails } from '../../HostDetailsSelectors';
+import SortableColumnHeaders from '../../../../Table/components/SortableColumnHeaders';
+
+const EnabledIcon = ({ streamText, streamInstallStatus, upgradable }) => {
```

Comment:
Moved these smaller components outside `ModuleStreamsTab` because they don't need to get recreated every render.